spec/features/medicaid_application_with_multiple_members_spec.rb

```diff
@@ -48,7 +48,9 @@
     end
 
     on_page "Introduction" do
-      expect(page).to have_content("Are you currently a US Citizen?")
+      expect(page).to have_content(
+        "Is everyone in your household currently a US citizen",
+      )
       click_on "Yes"
```

Comment:
Maybe we should say `no` here so we have a chance to indicate who is *not* a citizen in a feature spec?